utils/preprocessing.py

```diff
@@ -189,16 +210,21 @@ def preprocess_dataframe(df_name: str, df: pd.DataFrame) -> pd.DataFrame:
     return preprocessed_df
 
 
-def preprocess_data(data_dict: Dict) -> Dict:
+def preprocess_data(data_dict: Dict, config: DictConfig) -> Dict:
     """
     Parameters:
         data_dict: Dictionary with dataframes (each of which to be preprocessed differently)
+        config : Config file for variables
     Returns:
         Dictionary of preprocessed dataframes
     """
     processed_data_dict = {}
     logger.info('Preprocessing dataframes')
+    global year_initial, year_end
```

Comment:
please do not define global variables, just pass the variables that you need to the certain functions

Reply:
@cristian-castro-a, we have all 9 predictors in the preprocessing file where we need those variables and I believe since the global only needs to get values once we could eliminate editing constant parameters for 9 different functions in the same .py file. Let me know your thoughts.

Reply:
It can work, but at least to everything that I have read, using global variables is a really bad practice. You decide :)

Reply:
I would like to keep them, as global constants are accepted. :) But interesting to know variables are big NO.